svglib/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "svglib" %}
+{% set version = "0.8.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "1f8cb633f154b83987e3a906a55cf69c8739c0df5ea4eee024677c3e8fb62f55" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - pytest-runner
+    - setuptools
+    - reportlab
+    - lxml
+
+  run:
+    - python
+    - reportlab
+    - lxml
+
+test:
+
+  imports:
+    - svglib
+
+  requires:
+    - pytest
+
+  source_files:
+    - tests
+
+  commands:
+    # see deeplook/svglib#70 why skipping some tests on Windows
+    - pytest tests -k "not test_convert_pdf"  # [win]
+    - pytest tests  # [not win]
+
+about:
+  home: https://github.com/deeplook/svglib
+  license: LGPL
```

Comment:
v3, should I add `LGPL v3`? Btw, is there a resource somewhere that lists what should be put here, or is this field considered free form in general and in this case just missing some form of version specifier and `LGPL 3`,  `LGPL v3`, and `LGPL3` would all be acceptable?